service-print/src/main/java/org/oskari/print/PDF.java

```diff
@@ -321,6 +336,79 @@ private static void drawTimeseriesTexts(PDPageContentStream stream,
                 x, OFFSET_TIME_IN_TIMESERIES_BOTTOM);
     }
 
+    private static void drawCornerCoords(PDPageContentStream stream, PrintRequest request, float x, float y, float mapHeight, float mapWidth, PDRectangle pageSize)
+            throws IOException {
+
+        // get coordinates from bounding box
+        int[] coords = getCornerCoords(request);
+        
+        int topLon = coords[0];
+        int topLat = coords[3];
+        int bottomLat = coords[1];
+        int bottomLon = coords[2];
+        
+        drawTopCoords(stream, topLat, topLon, x, y, mapHeight, mapWidth);
+        drawBottomCoords(stream, bottomLat, bottomLon, x, y, mapHeight, mapWidth); 
+    }
+    
+    private static void drawTopCoords(PDPageContentStream stream, int topLat, int topLon, float x, float y, float mapHeight, float mapWidth)
+            throws IOException {
+        float x1 = x;
+        float y2 = mapHeight + y;
+        float x2 = x1 - OFFSET_CROSS_TOP;
+        float y1 = y2 + OFFSET_CROSS_TOP;
+        
+        drawCross(stream, x1, x2, y1, y2);
+        
+        float textX = x1 + OFFSET_TEXT_TOP;
+        float textY = y1 - OFFSET_TEXT_TOP;
+        float textY2 = textY - OFFSET_TEXT_Y;
+        
+        String coords = rb.getString("coordinates.upper") + ": " + topLat + ", " + topLon;
+        String projection = "ETRS-TM35FIN" + " " + rb.getString("coordinates");
+
+        PDFBoxUtil.drawText(stream, coords, PDPrintStyle.FONT, PDPrintStyle.FONT_SIZE_SCALE, textX, textY);
+        PDFBoxUtil.drawText(stream, projection, PDPrintStyle.FONT, PDPrintStyle.FONT_SIZE_SCALE, textX, textY2);
+    }
+    
+    private static void drawBottomCoords(PDPageContentStream stream, int bottomLat, int bottomLon, float x, float y, float mapHeight, float mapWidth)
+            throws IOException {
+        float x1 = mapWidth + x;
+        float y2 = y;
+        float x2 = x1 + OFFSET_CROSS_BOTTOM;
+        float y1 = y2 - OFFSET_CROSS_BOTTOM;
+        
+        drawCross(stream, x1, x2, y1, y2);
+        
+        float textX = x1 - OFFSET_TEXT_BOTTOM_X;
+        float textY = y1 + OFFSET_TEXT_BOTTOM_Y;
+        float textY2 = textY - OFFSET_TEXT_Y;
+        
+        String coords = rb.getString("coordinates.lower") + ": " + bottomLat + ", " + bottomLon;
+        String projection = "ETRS-TM35FIN" + " " + rb.getString("coordinates");
```

Comment:
We shouldn't hard-code "ETRS-TM35FIN". Get it from the messages based on projection with fallback to the EPSG-code.